src/services/dashboards/dashboard-create/DashboardCreatePage.vue

```diff
@@ -1,39 +1,76 @@
 <template>
     <div class="dashboard-create-page">
-        <p-heading
-            show-back-button
-            :title="$t('DASHBOARDS.CREATE.TITLE')"
-            @click-back-button="$router.go(-1)"
-        />
-        <section class="dashboard-create-form-container">
+        <div v-if="currentStep === steps[0].step"
+             class="dashboard-create-step1"
+        >
+            <dashboard-create-header
+                :description="steps[0].description"
+                :total-steps="steps.length"
+                :current-step="currentStep"
+            />
             <dashboard-scope-form :dashboard-scope.sync="dashboardScope"
                                   @set-project="setForm('dashboardProject', $event)"
             />
-            <dashboard-template-form @set-template="setForm('dashboardTemplate', $event)" />
             <dashboard-viewer-form :dashboard-viewer-type.sync="dashboardViewerType" />
-        </section>
-        <div class="dashboard-create-buttons">
-            <p-button style-type="tertiary"
-                      size="lg"
-                      @click="$router.go(-1)"
-            >
-                {{ $t('DASHBOARDS.CREATE.CANCEL') }}
-            </p-button>
-            <p-button style-type="primary"
-                      size="lg"
-                      :disabled="!isAllValid"
-                      @click="handleClickCreate"
-            >
-                {{ $t('DASHBOARDS.CREATE.CREATE') }}
-            </p-button>
+            <div class="button-area">
+                <p-button
+                    style-type="transparent"
+                    size="lg"
+                    @click="$router.go(-1)"
+                >
+                    Cancel
+                </p-button>
+                <p-button
+                    style-type="primary"
+                    size="lg"
+                    :loading="false"
+                    :disabled="!valid"
+                    @click="handleGoStep(2)"
+                >
+                    Continue
+                </p-button>
+            </div>
+        </div>
+        <div v-if="currentStep === steps[1].step"
+             class="dashboard-create-step2 max-w-lg2"
+        >
+            <dashboard-create-header
+                :description="steps[1].description"
+                :total-steps="steps.length"
+                :current-step="currentStep"
+            />
```

Comment:
Suggestion: How about combining it with the header in step 1?
```suggestion
<dashboard-create-header
                :description="steps[currentStep].description"
                :total-steps="steps.length"
                :current-step="currentStep"
            />
```